test/functionalTest/cases/3162_context_providers_legacy_non_primitive/registration_without_provider.test

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This test is under directory 3162_context_providers_legacy_non_primitive/ but it seem is has nothing to do with CP... maybe should be moved to another directory? Maybe is already covered (maybe grep for "geo:box" could help)?

Reply:
The test is called registration_without_provider.test ...

But, what happened to the registration?   It's gone.
I guess I was experimenting and forgot to revert it.

Once t he registration is back, you will see the connection to CPs.